doc/sphinx-guides/source/installation/config.rst

```diff
@@ -781,7 +781,7 @@ Basic Database Settings
      - | ``dataverse``
        | (installer sets to ``dvndb``)
    * - dataverse.db.parameters
-     - Connection parameters, such as ``sslmode=require``. See `Postgres JDBC docs <https://jdbc.postgresql.org/documentation/head/connect.html>`_
+     - Connection parameters, such as ``sslmode=require``. See `Postgres JDBC docs <https://jdbc.postgresql.org/documentation/head/connect.html>`
```

Comment:
The _ suffix may not be removed here. In RestructuredText links need this char to be declared links, otherwise the backticks are interpreted as layouting directives. See also the preview generated at https://dataverse-guide--10670.org.readthedocs.build/en/10670/installation/config.html#basic-database-settings